### ISSUE TITLE
Add integration with Apple COVID-19 notifications (not contact tracing)

### ISSUE THREAD

**Issue by kaplanlior** [MohGovIL/hamagen-react-native#284]
See https://9to5mac.com/2020/09/01/covid-19-exposure-ios-13-7-built-in/

( opened based on https://www.facebook.com/groups/hitechproblems/permalink/1102128160189572/ )

**Comment by lgariv** [comment]
Exposure Notifications Express is not meant to be a part of an app tho. The whole purpose of it is to be accessible to as many people as possible, without having to install an app (correct me if I'm wrong). It can still be developed as a separate "project", and it **is** a really good thing to have. 

Also, it would probably be a good idea to change the title to `Add integration with Exposure Notifications Express`, as Google will soon add support for this as well for Android 6.0 and up [(read that)](https://www.google.com/amp/s/www.cnet.com/google-amp/news/apple-google-promise-better-coronavirus-tracking-with-exposure-notifications-express/); Removing the `(not contact tracing)` part, as this is, in fact, contact tracing. It is also stated in the link from 9to5mac that @kaplanlior provided.

**Comment by DorHersh** [comment]
This really need to reconsidered ;
I just got a notification asking me if I was in Dizzingoff Center 10 days ago from 16:00(till infinity as there was no to__).
As you all know - Dizzingoff center is a huge place and GPS approach doesn’t help indoors..
(I do have Bluetooth on for Hamagen but I just doesn’t help.

**Comment by lgariv** [comment]
> This really need to reconsidered ;
> 
> I just got a notification asking me if I was in Dizzingoff Center 10 days ago from 16:00(till infinity as there was no to__).
> 
> As you all know - Dizzingoff center is a huge place and GPS approach doesn’t help indoors..
> 
> (I do have Bluetooth on for Hamagen but I just doesn’t help.

I agree. GPS is not accurate enough; regarding Bluetooth, it just won't make any change even if implemented in a better way, unless more people will download the app. Most people don't even know that this app exists, some don't like the use of GPS (even though everything is happening on-device) and unfortunately some people just don't care enough to download it.

Exposure Notifications Express solves all of this problems, by making it easily available for everyone without having to download an app, and taking advantage of the Apple/Google API. I believe that using Exposure Notifications Express, combined with raising awareness and educating the public about what it is and how to make sure it is turned on, **will** make a positive impact.

**Comment by lgariv** [comment]
Google has created an [implementation of a key server](https://github.com/google/exposure-notifications-server/) that can be used as a starting point for creating your own key server. We can [configure](https://developer.apple.com/documentation/exposurenotification/supporting_exposure_notifications_express/configuring_a_key_server_for_exposure_notifications_express) it to be used for Exposure Notification Express. 

Than, we have to [create an Exposure Notification Express test verification server](https://developer.apple.com/documentation/exposurenotification/supporting_exposure_notifications_express/setting_up_an_exposure_notifications_express_test_verification_server) - which [Google also implemented](https://github.com/google/exposure-notifications-verification-server) to be used as a starting point.

**Comment by lgariv** [comment]
> Google has created an [implementation of a key server](https://github.com/google/exposure-notifications-server/) that can be used as a starting point for creating your own key server. We can [configure](https://developer.apple.com/documentation/exposurenotification/supporting_exposure_notifications_express/configuring_a_key_server_for_exposure_notifications_express) it to be used for Exposure Notification Express. 
> 
> Than, we have to [create an Exposure Notification Express test verification server](https://developer.apple.com/documentation/exposurenotification/supporting_exposure_notifications_express/setting_up_an_exposure_notifications_express_test_verification_server) - which [Google also implemented](https://github.com/google/exposure-notifications-verification-server) to be used as a starting point. 

@kaplanlior @MiriShalev 
Is there any progress on it?
I can probably help to make a key server. That's pretty much 90% of the work as far as I understand it

**Comment by lgariv** [comment]
@kaplanlior @MiriShalev 
Anything?

**Comment by kaplanlior** [comment]
Hi @lgariv,

No update at the moment. I'll post here if there's a change.

Thanks for caring!

**Comment by lgariv** [comment]
> Hi @lgariv,
> 
> 
> 
> No update at the moment. I'll post here if there's a change.
> 
> 
> 
> Thanks for caring!

Of course! Btw, I tried to make a private testing server in the past 2 weeks. It's a fairly simple and fully automated process, really user friendly (for the developer, that is).

The only thing that needs to be done manually, at least of the initial setup stage, is creating and configuring a gcloud account (Google Cloud), which is also pretty simple in their online GUI.

**Comment by lgariv** [comment]
@kaplanlior It's been over 3 months now... So I guess this is not gonna happen?

**Comment by kaplanlior** [comment]
> > Hi @lgariv,
> > No update at the moment. I'll post here if there's a change.
> > Thanks for caring!
> 
> Of course! Btw, I tried to make a private testing server in the past 2 weeks. It's a fairly simple and fully automated process, really user friendly (for the developer, that is).

I wouldn't expect it, see https://www.kan.org.il/item/?itemId=80743

**Comment by lgariv** [comment]
> > > Hi @lgariv,
> > > No update at the moment. I'll post here if there's a change.
> > > Thanks for caring!
> > 
> > 
> > Of course! Btw, I tried to make a private testing server in the past 2 weeks. It's a fairly simple and fully automated process, really user friendly (for the developer, that is).
> 
> I wouldn't expect it, see https://www.kan.org.il/item/?itemId=80743

Exposure Notification Express is not something that's integrated in an application. It is just a server that Google's/Apple's framework (that is already built into Android and iOS, so the user won't have to download an app) is using. It does not have anything to do with an app, so...
